### ISSUE TITLE
Add option to enable all presets from original puzzles

### ISSUE THREAD

**Issue by tmp6154** [chrisboyle/sgtpuzzles#340]
Hello!

I've noticed that some games don't include all presets from original puzzle collection, particularly, Mines include only few of original presets. While there's still "Custom..." option available, it still a little sad. I understand, that most of excluded presets are too large and inconvenient to be played on smartphone screen, but they can be perfectly played on tablets. I love playing Mines on higher difficulties, but they're not available, except via manually entering via Custom option.

I propose a solution to add an option to settings that will show all presets, including ones with large boards.

**Comment by chrisboyle** [comment]
Probably I could achieve better defaults if they were Android array resources, with at least two variants according to screen size. Then the option to show all could still be implemented but it would only need to be on the Advanced settings screen, as most people would have a sensible selection already.

**Comment by tmp6154** [comment]
Yeah, sounds like a great idea! Would allow both choice and adaptivity. In that case, option could be tristate, like night mode option - "adaptive", "force all presets" and "force small presets" with "adaptive" as default. "Force small presets" might make sense, too, for people with bad eyesight so that they always have only choice of presets that would be large, even for large screen.
